akita_inu_asa_utils.py

```diff
@@ -192,7 +198,8 @@ def create_app_signed_txn(private_key,
                                                     clear_program,
                                                     global_schema,
                                                     local_schema,
-                                                    app_args)
+                                                    app_args,
+                                                    **kwargs)

```

Comment:
If foreign_assets is the only additional arg here then maybe just add that instead

Reply:
This makes sense, and would fit the style established by some of the other functions in the module. Will do.